Solutions/DigitalGuardianDLP/Hunting Queries/DigitalGuardianRareUrls.yaml

```diff
@@ -0,0 +1,25 @@
+id: b9a69da9-1ca0-4e09-a24f-5d88d57e0402
+name: Digital Guardian - Rare Urls
+description: |
+  'Query searches for rare Urls.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: DigitalGuardianDLP
+    dataTypes:
+      - DigitalGuardianDLPEvent
+tactics:
+  - Exfiltration
+relevantTechniques:
+  - T1048
+query: |
+  DigitalGuardianDLPEvent
+  | where TimeGenerated > ago(24h)
+  | where isnotempty(http_url)
+  | summarize count() by SrcUserName, http_url
+  | order by count_ asc

```

Comment:
Wondering if we wanted to use top 5 or top 10 here to present rare URL.

Reply:
fixed